src/metabase/models/setting.clj

```diff
@@ -370,11 +370,14 @@
 
 (defn- user-facing-info [setting]
   (let [k (:name setting)
-        v (get k)]
+        v (get k)
+        set-by-env (not (nil? (env-var-value setting)))]
```

Comment:
in idiomatic Clojure you'd name this something like `set-by-env?` to indicate that it's a boolean value